vhost-device-sound/CHANGELOG.md

```diff
@@ -9,6 +9,20 @@
 
 ### Deprecated
 
+## v0.1.1
```

Comment:
I was thinking whether we should release v0.2.0, but maybe we haven't changed anything from the user's point of view, so I think v.0.1.1 is fine. Opinions?

Reply:
The only user facing change here seems to be the `pipewire` dependency; does it require updated system dependencies for the end-user? If yes, a `0.2.0` would make sense I think.